Unable to build R-modular interface

Hi,

I tried to build the R-modular interface but make failed 

> [100%] [100%] Building CXX object src/interfaces/r_modular/CMakeFiles/r_modular.dir/sg_print_functions.cpp.o
Building CXX object src/interfaces/r_modular/CMakeFiles/r_modular.dir/modshogunR_wrap.cxx.o
In file included from /home/bee/opt/hdf5/include/hdf5.h:24:0,
                 from /home/bee/Downloads/shogun-develop/src/shogun/io/HDF5File.h:19,
                 from /home/bee/Downloads/shogun-develop/build/src/interfaces/r_modular/modshogunR_wrap.cxx:3085:
/home/bee/opt/hdf5/include/H5public.h:198:33: warning: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wliteral-suffix]
 #   define H5_PRINTF_HADDR_FMT  "%"H5_PRINTF_LL_WIDTH"u"
                                 ^
Linking CXX shared module modshogun.so
Generating modshogun.RData
 f="modshogun.R"; fdata="modshogun.RData"; source ( f ) ; save ( list=ls ( all=TRUE ) ,file=fdata, compress=TRUE ) ; q ( save="no" ) 
Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection
Calls: source -> file
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
  cannot open file 'modshogun.R': No such file or directory
Execution halted
make[2]: *** [src/interfaces/r_modular/modshogun.so] Error 1
make[1]: *** [src/interfaces/r_modular/CMakeFiles/r_modular.dir/all] Error 2
make: *** [all] Error 2

Tried both shogun 3.2.0 and developement version. R-static interface built fine though. 

R version 3.1.2, OS is Ubuntu 14.04 64 bit.

Thanks for help in advance.

--- a/src/interfaces/r_modular/CMakeLists.txt
+++ b/src/interfaces/r_modular/CMakeLists.txt
@@ -6,7 +6,7 @@ set_target_properties(r_modular PROPERTIES PREFIX "")
 
 ADD_CUSTOM_COMMAND(TARGET r_modular
 	POST_BUILD
-	COMMAND echo 'f="modshogun.R" \; fdata="modshogun.RData" \; source( f ) \; save( list=ls( all=TRUE ) , file=fdata , compress=TRUE ) \; q( save="no" ) \;' | ${R_EXECUTABLE} --silent --no-save
+	COMMAND echo 'wd="${CMAKE_CURRENT_BINARY_DIR}" \; setwd( wd ) \; f="modshogun.R" \; fdata="modshogun.RData" \; source( f ) \; save( list=ls( all=TRUE ) , file=fdata , compress=TRUE ) \; q( save="no" ) \;' | ${R_EXECUTABLE} --silent --no-save
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 	COMMENT "Generating modshogun.RData"
 )

